### DISCUSSION TITLE
Why `openConsole` and `openDevtools` do not work?

### DISCUSSION THREAD

**Discussion by ayub-kokabi** [wxt-dev/wxt#811] [Q&A] [answered]
```typescript
// web-ext.config.ts:
import { defineRunnerConfig } from "wxt";

export default defineRunnerConfig({
  startUrls: ["https://info.cern.ch/"],
  openConsole: true,
  openDevtools: true
});
```

When I run `pnpm dev`, the browser opens with the specified `startUrl`, but `openConsole` and `openDevtools` do not work. I'm unsure if this is a bug with `web-ext` or if I missed something.

> Chrome Version 126.0.6478.127 (Official Build) (64-bit)

**Comment by aklinker1** [comment, marked as answer]
I believe those options are only supported by firefox

**Reply by ayub-kokabi** [reply]
Yes, you are right.
https://github.com/mozilla/web-ext/issues/2580

**Reply by ayub-kokabi** [reply]
> I believe those options are only supported by firefox

Is there an alternative to web-ext that supports both Chrome and Firefox well?

**Reply by aklinker1** [reply]
Not that I'm aware of

**Comment by supfiger** [comment]
https://wxt.dev/api/reference/wxt/interfaces/ExtensionRunnerConfig.html#chromiumargs
```js
chromiumArgs: [
  --auto-open-devtools-for-tabs /* https://peter.sh/experiments/chromium-command-line-switches/#auto-open-devtools-for-tabs */
],
```

If I am not wrong, you can achieve almost what you want by this argument in your WXT config.

Just use this arg for Chromium + also use `openDevtools` for Firefox.

**Reply by ayub-kokabi** [reply]
Thank you, it works. 🙏
